docs/_basic/ja_listening_modals.md

```diff
@@ -6,11 +6,14 @@ order: 12
 ---
 
 <div class="section-content">
-<a href="https://api.slack.com/reference/block-kit/views">モーダルのペイロード</a>が入力のブロックを含む場合、その入力値を受け取るために `view_submission` のリクエストをリッスンする必要があります。`view_submission` リクエストをリッスンするためには、組み込みの `view()` メソッドを使用します。
 
-`view()` メソッドは、文字列型または `RegExp` 型 の `callback_id` を必要とします。
+`view` メソッドを使ってユーザーからのインタラクションをリッスンすることができます。
```

Comment:
It seems that interaction here is the one with views. The current translation does not include the part.
```suggestion
`view` メソッドを使うと、ユーザーのビューとのインタラクションをリッスンすることができます。
```